COMPILING.md

```diff
@@ -4,7 +4,7 @@ COMPILING
 To compile Cuberite from source, you need the following set of software:
 
  * CMake
- * Platform-specific make tool \(windows would be MSVC, linux/osx GNU make, etc.\)
+ * Platform-specific make tool \(windows would be MSVC, linux/macos GNU make, etc.\)
```

Comment:
I think @sphinxc0re is referring to the capitalization: `macOS` instead of `macos`.